modules/core/keeper/msg_server_test.go

```diff
@@ -436,6 +470,7 @@ func (suite *KeeperTestSuite) TestHandleTimeoutPacket() {
 		packet    channeltypes.Packet
 		packetKey []byte
 		path      *ibctesting.Path
+		noop      bool // indicate no-op
```

Comment:
same suggestion about adding this to the test case rather than floating bool

Reply:
done